styles.css

```diff
@@ -6,22 +6,55 @@
   list-style: none;
 }
 
+.btn-see-project {
+  color: #396df2;
+  text-decoration: none;
+  letter-spacing: 0.03em;
+  font-weight: 500;
+  font-size: 17px;
+  text-transform: capitalize;
+  border: 1px solid #6070ff;
+  padding: 0.5em;
+  border-radius: 10px;
+  position: relative;
+  margin-left: 1.5rem;
+  margin-bottom: 5rem;
+}
```

Comment:
- [OPTIONAL] According to the requirements you should implement the button interactions (enable, hover, etc.) based on your Figma design, I marked it as optional but I strongly recommend implementing it 🙏💯

![image](https://user-images.githubusercontent.com/45231276/185756512-47271e92-3f6c-43ed-9f5a-ef2f7049c0f5.png)

![image](https://user-images.githubusercontent.com/45231276/185756426-4663aa31-8e53-4ba3-a982-459eb4866dec.png)